net/interfaces/interfaces_linux.go

```diff
@@ -230,3 +231,55 @@ func defaultRouteInterfaceAndroidIPRoute() (ifname string, err error) {
 	}
 	return ifname, nil
 }
+
+// AWS Lambda's only connectivity is via IPv4 link-local addresses statically
```

Comment:
start with documenting what it does in CommentSentences form: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences